google-cloud-testing/storage-benchwrapper/src/main/proto/storage.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 Google LLC
```

Comment:
sorry, missed this earlier - year should be 2019

Reply:
Done